clients/extension/src/components/middle-truncate/index.tsx

```
@@ -51,11 +56,12 @@ const Component: FC<ComponentProps> = ({ text }) => {
       truncating: true,
     }))
   }
+
   useEffect(ellipsisDidUpdate, [ellipsis, counter, text])
   useEffect(componentDidUpdate, [text])
 
   return (
-    <span ref={elmRef} className="middle-truncate">
+    <span ref={elmRef} onClick={handleClick} className="middle-truncate">
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)